tests/test_phenicx_anechoic.py

```diff
@@ -0,0 +1,304 @@
+import os
+import shutil
+import numpy as np
+import pytest
+
+from mirdata.datasets import phenicx_anechoic
+from mirdata import annotations, download_utils
+from tests.test_utils import run_track_tests, run_multitrack_tests
+
+
+def test_track():
+    default_trackid = "beethoven-violin"
+    data_home = "tests/resources/mir_datasets/phenicx_anechoic"
+    dataset = phenicx_anechoic.Dataset(data_home)
+    track = dataset.track(default_trackid)
+
+    expected_attributes = {
+        "track_id": "beethoven-violin",
+        "audio_paths": [
+            "tests/resources/mir_datasets/phenicx_anechoic/"
+            + "audio/beethoven/violin1.wav",
+            "tests/resources/mir_datasets/phenicx_anechoic/"
+            + "audio/beethoven/violin2.wav",
+            "tests/resources/mir_datasets/phenicx_anechoic/"
+            + "audio/beethoven/violin3.wav",
+            "tests/resources/mir_datasets/phenicx_anechoic/"
+            + "audio/beethoven/violin4.wav",
+        ],
+        "notes_path": "tests/resources/mir_datasets/phenicx_anechoic/"
+        + "annotations/beethoven/violin.txt",
+        "notes_original_path": "tests/resources/mir_datasets/phenicx_anechoic/"
+        + "annotations/beethoven/violin_o.txt",
+        "instrument": "violin",
+        "piece": "beethoven",
+        "n_voices": 4,
+    }
+
+    expected_property_types = {
+        "notes": annotations.NoteData,
+        "notes_original": annotations.NoteData,
+        "audio": tuple,
+    }
+
+    run_track_tests(track, expected_attributes, expected_property_types)
```

Comment:
should we write a similar function for `run_multitrack_tests`? We might even be able to use the same, just replacing `track` by `multitrack` in the function call.

Reply:
nevermind! I just saw you already did it :)